examples/cli/discovery.md

```diff
@@ -176,9 +176,9 @@ You can see that the two instances of `torch.nn.Linear`, `fc` and `fc2`, are ben
 
```

Comment:
Line 173 See the [Max Depth documentation](https://github.com/groq/mlagility/blob/main/tools_user_guide.md#maximum-analysis-depth) for more details.

Reply:
got it!